### PR TITLE
Oep 18

### PR DESCRIPTION
**Pull request by estute**
As part of the python 2 -> python 3 upgrade, make this repo oep-18 compliant so that it's easier to update libraries in the sandboxed environments.

### CODE REVIEW

**Comment by estute** [comment]
jenkins test python 2

File: Makefile [hunk 1]
```diff
@@ -15,3 +15,12 @@ test_no_proxy:
 test_proxy:
 	@echo "Running all tests with proxy process"
 	CODEJAIL_PROXY=1 nosetests --with-xunit --xunit-file reports/nosetests-proxy.xml
+
+
+export CUSTOM_COMPILE_COMMAND = make upgrade
```
Comment (feanil) [inline Makefile]: Shouldn't this be like this: https://github.com/edx/edx-platform/blob/master/Makefile#L68

Comment (estute) [reply Makefile]: fixed

**Comment by estute** [comment]
jenkins test python 2

**Comment by estute** [comment]
jenkins test python 2

**Pull request merged**